testnets/selfchaindevnet/assetlist.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "selfchaindevnet",
+  "assets": [
+    {
+      "description": "Self Chain Is the First Modular Intent-Centric Access Layer L1 Blockchain and Keyless Wallet Infrastructure Service That Uses MPC-TSS/AA for Multi-Chain Web3 Access.",
```

Comment:
There also exists a Description property in chain.json, which this feel more appropraite for.